daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala

```diff
@@ -379,20 +381,21 @@ object ValueGenerators {
       key <- Gen.option(keyWithMaintainersGen)
       byKey <- Gen.oneOf(true, false)
     } yield Node.Exercise(
-      targetCoid = targetCoid,
-      templateId = templateId,
-      choiceId = choiceId,
-      consuming = consume,
-      actingParties = actingParties,
-      chosenValue = chosenValue,
-      stakeholders = stakeholders,
-      signatories = signatories,
+      targetCoid,
+      templateId,
+      interfaceId,
+      choiceId,
+      consume,
+      actingParties,
+      chosenValue,
+      stakeholders,
+      signatories,
       choiceObservers = choiceObservers,
-      children = children,
-      exerciseResult = exerciseResult,
-      key = key,
-      byKey = byKey,
-      version = version,
+      children,
+      exerciseResult,
+      key,
+      byKey,
+      version,
```

Comment:
can we stick to `x=x` (`key = key`, `byKey = byKey`, `version = version`, etc) here? it's so easy to mess up the order for positional arguments.

Reply:
reverted